common/common-javassist/src/main/java/org/apache/servicecomb/common/javassist/JavassistUtils.java

```diff
@@ -185,10 +185,47 @@ private static void addEnumValuesMethod(CtClass ctClass, List<String> values) th
         }
       }
 
-      LOGGER.info("generate {} in classLoader {}.", config.getClassName(), classLoader);
-      return ctClass.toClass(classLoader, null);
+      LOGGER.info("create CtClass {} in classLoader {}.", config.getClassName(), classLoader);
+      return ctClass;
+    } catch (Throwable e) {
+      throw new IllegalStateException(
+          String.format("Failed to create CtClass %s in classLoader %s.", config.getClassName(),
+              classLoader), e);
+    }
+  }
+
+  public static Class<?> createClass(ClassLoader classLoader, ClassConfig config) {
+    if (classLoader == null) {
```

Comment:
It's better to add a log here for tracking the classloader issue.

Reply:
i have test for this
because already throw exception with error context, invoker already print the exception
if print in this place, then will print multi times.

so i think it's better not log here.
what's your suggestion?

Reply:
oh, my suggestion is not print the exception here , it just write some log message for switching  the classloader to  thread context classloader.

Reply:
Whether success or failure， will print the class belongs to which classLoader
so that no need to print this log.